src/main/scala/io/qbeast/core/model/QDataType.scala

```diff
@@ -72,31 +72,49 @@ trait OrderedDataType extends QDataType {
   @JsonIgnore
   val ordering: Numeric[Any]
 
+  val defaultQuantiles: IndexedSeq[Any]
+
 }
 
 object DoubleDataType extends OrderedDataType {
   override def name: String = "DoubleDataType"
   override val ordering: Numeric[Any] = implicitly[Numeric[Double]].asInstanceOf[Numeric[Any]]
+
+  override val defaultQuantiles: IndexedSeq[Any] =
+    BigDecimal(Double.MinValue).to(BigDecimal(Double.MaxValue), BigDecimal(1e307)).map(_.toDouble)
+
 }
 
 object IntegerDataType extends OrderedDataType {
   override def name: String = "IntegerDataType"
   override val ordering: Numeric[Any] = implicitly[Numeric[Int]].asInstanceOf[Numeric[Any]]
+  override val defaultQuantiles: IndexedSeq[Any] = Int.MinValue.to(Int.MaxValue)
```

Comment:
Paola, be careful, this method is creating a list with 2^32 elements. When it gets serialized into the metadata, it would be huge.